src/arrays.js

```diff
@@ -1,61 +1,70 @@
 const getNthElement = (index, array) => {
-  // your code here
+  if (index >= array.length) {
+    return array[index - array.length];
+  }
+  return array[index];
```

Comment:
This works for the tests provided. However ,if the index is greater than 2 times the length of the array this will return the wrong result. A better solution is to use the modulus operator `%`. e.g.
```
return array[index % array.length]
```
Try having a play around with this with different indexes/array lengths to see if you can work out how/why this works